test/Validation.test.js

```diff
@@ -174,6 +175,65 @@ describe("Validation", function() {
 			"    * configuration.devtool should be a string.",
 			"    * configuration.devtool should be false"
 		]
+	}, {
+		name: "non-absolute path",
+		config: {
+			entry: "foo.js",
+			output: {
+				filename: "/bar"
+			}
+		},
+		message: [
+			" - configuration.output.filename A non-absolut path is expected. However the provided value \"/bar\" is an absolute path!",
+			"    ({",
```

Comment:
This should not be printed.

Add you addition to lib/WebpackOptionsValidationError.js (formatSchema and formatValidationError)